data-sources/vercel/config.yml

```
@@ -0,0 +1,34 @@
+id: vercel
+displayName: Vercel
+description: |
+  ## About Vercel
+
+  Vercel is the platform for frontend developers, providing the speed and reliability innovators need to create at the moment of inspiration.
+
+  We enable teams to iterate quickly and develop, preview, and ship delightful user experiences. Vercel has zero-configuration support for 35+ frontend frameworks and integrates with your headless content, commerce, or database of choice.
+  
+  ## About this quickstart
+
+  This quickstart is the fastest way to jump straight into monitoring your end-to-end development logs.
+icon: logo.svg
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId: fc2bb0ac-6622-43c6-8c1f-6a4c26ab5434
```

This is the data source related to the "Vercel Logs" quickstart, correct?
If so, this quickstart id should be `c18372e0-b849-408f-b2b1-3ea7096ef5a8`
```suggestion
        quickstartId: c18372e0-b849-408f-b2b1-3ea7096ef5a8
```